Client/src/index.css

```diff
@@ -54,6 +54,12 @@ html {
 
 	--env-var-shadow-1: 0px 4px 24px -4px rgba(16, 24, 40, 0.08),
 		0px 3px 3px -3px rgba(16, 24, 40, 0.03);
+
+	/* ThemeSwitch Component Stylings */
+	--ease-elastic-3: cubic-bezier(0.68, -0.55, 0.27, 1.55);
+	--ease-elastic-4: cubic-bezier(0.68, -0.55, 0.27, 1.55);
+	--ease-3: cubic-bezier(0.25, 0.1, 0.25, 1);
+	--ease-out-5: cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
@peterpardo we aren't going to use these anywhere else in the application, can you move them to the ThemeSwitch specific CSS file?  Thanks

Reply:
Ok, let me work on this real quick.

Reply:
Here is the commit for this change: https://github.com/bluewave-labs/checkmate/pull/1409/commits/b97e70f1629b2083f74f8e16c33124afb290b454. 

I directly used the styling inside the `ThemeSwitch` css file since it will only be used there. Thanks!